File: 5week/jungmin/javascript/0109_배열회전시키기.js
function solution(numbers, direction) {
    var arr = [];
    var a = numbers[numbers.length-1];
    var b = numbers[0];
    if (direction === "right") {
        arr.push(a);
        for(i=0; i<numbers.length-1; i++){
            arr.push(numbers[i]);
        }
        return arr;
    } else {
        for(i=1; i<numbers.length; i++){
            arr.push(numbers[i]);
        }
        arr.push(b);
        return arr;
    }
}

File: 5week/jihwan/python/0109_배열회전시키기.py
# 프로그래머스 Level 0 0109_배열회전시키기
# https://school.programmers.co.kr/learn/courses/30/lessons/12982

def solution(numbers, direction):
    answer = []
    
    if(direction == 'right'):              
            answer.append(numbers[-1])
            answer += numbers[0:-1]
    if(direction == 'left'):
            answer += numbers[1:]
            answer.append(numbers[0])
                           
    return answer
